requirements/common_constraints.txt

```diff
@@ -0,0 +1,37 @@
+
+# A central location for most common version constraints
+# (across edx repos) for pip-installation.
+#
+# Similar to other constraint files this file doesn't install any packages.
+# It specifies version constraints that will be applied if a package is needed.
+# When pinning something here, please provide an explanation of why it is a good
+# idea to pin this package across all edx repos, Ideally, link to other information
+# that will help people in the future to remove the pin when possible.
+# Writing an issue against the offending project and linking to it here is good.
+#
+# Note: Changes to this file will automatically be used by other repos, referencing
```

Comment:
You should not download this file directly, just reference it from edx-lint like we do here: https://github.com/openedx/docs.openedx.org/blob/main/requirements/constraints.txt